src/Components/Grants/GrantsShow.jsx

```diff
@@ -217,13 +219,18 @@ export default function GrantsShow() {
           breadCrumbLink={`/organizations/${currentOrganization.id}/grants/`}
           copyLink={`/grants/${grant.id}/copy/`}
           editLink={`/grants/${grant.id}/edit/`}
+          setOverView={setOverView}
+          overView={overView}
         />
-        <Container
-          className="grants-show__sections-container"
-          as="section"
-          centered
-        >
-          {/* <DndContext
+        {overView ? (
+          <GrantShowOverview grant={grant} />
```

Comment:
wyt of making this a separate route instead of conditionally rendering it in here? something like `/organizations/:organizationId/grants/:grantId/reorder`

i only ask because I think it might help keep this component a little smaller and simpler but lmk!

Reply:
We were running into a bunch of problems with the way the data was being passed around - for now, it seems like it might be simpler to build it in here and then create a separate endpoint later. GrantShow is definitely very long....